core/base/src/TEnv.cxx

```diff
@@ -385,7 +385,7 @@ TString TEnvRec::ExpandValue(const char *value)
    return val;
 }
 
-ClassImp(TEnv);
+ClassImp(TEnv)
```

Comment:
What error/warning is the semi-colon removal fixing?

Note that there is a side-effect at least on code editing, see commits

5fe56a2e3215888f709b0b032b572b0c136f9201
and
973ba9c87b28a139eb5ce825a0763f90121bb472

Reply:
I think it was motivated by some warnings of the type:
`warning: Found ';' while parsing initializer list!`
but here, if I recall correctly, I think I did out of consistency with others that I had removed nearby, not because of a warning right there. I can revert it.

Maybe we should file instead a bug report to doxygen if we find that this warning is non-sense.

Reply:
> Maybe we should file instead a bug report to doxygen if we find that this warning is non-sense.

Please :)

Reply:
Reverted